client/src/styles/home.css

```diff
@@ -0,0 +1,33 @@
+.home__banner__wrapper {
+  margin: 30px;
+  top: 20%;
+  position: absolute;
```

Comment:
why placing it absolute and not using flex to have it always in the middle of the page on every display size for example?